ext/mysqli/mysqli.stub.php

```diff
@@ -845,9 +845,9 @@ public function get_server_info(): string {}
     public function get_warnings(): mysqli_warning|false {}
 
     /**
-     * @deprecated
      * @return bool|null
      * */
+    #[\Deprecated(since: '8.1', message: 'replace calls to parent::init() with parent::__construct()')]
```

Comment:
why do think people call this method via `parent::`? Wouldn't it be better to just reference the real class name instead?

Reply:
I took this from the documentation changelog. If the real class should be referenced then the docs should be fixed too. But remember a class extending mysqli could also be extended, and to have a proper initialization sequence you should call every single class initializer